### ISSUE TITLE
Multiple notes in Notepad

### ISSUE THREAD

**Issue by NiharikaVadlamudi** [akshgpt7/MarvellOS#20]
We need to figure out where the text is being saved & a button to go back to Home Page . 
@akshgpt7 I would work on these tasks , pls assign it to me

**Comment by akshgpt7** [comment]
> We need to figure out where the text is being saved 

I don't really understand this. The save button means that it'll persist the text for the next time you open the notepad.

As for the Home button, it does exist on the notepad window (maybe not visible due to your window size?), If that's the case, you can try making the window bigger through the code and see if the button's there.

**Comment by NiharikaVadlamudi** [comment]
" We need to figure out where the text is being saved " ---- So, I wanted to make sure that the text is saved after you click the SAVE button , and a new notepad comes up on the screen. (Multiple notes)  
Also, the GUI is not expandable in nature hence the button is not visible to me. I'll change the aesthetics and interface part of it properly .

**Comment by akshgpt7** [comment]
Yes, I'm aware of the dicey positioning and placement of various elements, and there's an issue for it (#7). 
As for the save, do you want to create something like multiple notes, that one can choose from? 
Sounds like a good idea (try to implement something like they're able to give names to that note as well).

**Comment by akshgpt7** [comment]
Also, since the note is being saved in a file, please try and have just one file to save them all (maybe through some sort of separator), or limit the maximum number of notes to five, so we have 5 separate files to store each note.
